sdks/go/container/boot.go

```diff
@@ -137,7 +137,9 @@ func main() {
 		"--logging_endpoint=" + *loggingEndpoint,
 		"--control_endpoint=" + *controlEndpoint,
 		"--semi_persist_dir=" + *semiPersistDir,
-		"--options=" + options,
+	}
+	if err := tools.MakePipelineOptionsFileAndEnvVar(options); err != nil {
+		logger.Fatalf(ctx, "Failed to load pipeline options to worker: %v", err)
```

Comment:
For my education, when this error is triggered, Dataflow job fails immediately?

Reply:
It should, but I'm not actually sure since it depends on Dataflow's container observation semantics.